### DISCUSSION TITLE
[Esercitazione 13 Dicembre 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#320] [Esercizi - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/fe8b5f58-92e7-4569-9fbd-c208f8af70a2)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/abc425d6-e204-4729-aae1-a08af1f3ce3e)

**Reply by Alby7503** [reply]
Non mi tornava il conto su riga 3 colonna 2 perciò l’ho rifatto a mano ed ho anche provato online e credo debba essere 2 non 0

<img width="431" alt="image" src="https://github.com/sapienzastudentsnetwork/algebra2324/assets/44052142/ec3b12ef-1f75-4e0a-a111-eec8c3811fd5">

**Reply by CiottoloMaggico** [reply]
Si hai ragione

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/e63f7ece-9d3b-4240-a746-465814c9e4ad)
